### PR TITLE
Updating Fieldset UiComponent documentation

### PR DESCRIPTION
**Pull request by eduard13**
## Purpose of this pull request

This pull request (PR) updates the Fieldset UiComponent documentation.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/ui_comp_guide/components/ui-fieldset.html
- https://devdocs.magento.com/guides/v2.2/ui_comp_guide/components/ui-fieldset.html

## Links to Magento source code

-  https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/Email/view/adminhtml/ui_component/design_config_form.xml#L9
- https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/SalesRule/view/adminhtml/ui_component/sales_rule_form.xml#L58

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.2/ui_comp_guide/components/ui-fieldset.md [hunk 1]
```diff
@@ -20,8 +20,46 @@ The Fieldset component implements a container for visually-grouped form elements
 | Title         | Type    | Default | Description                                                                       |
 | ------------- | ------- | ------- | --------------------------------------------------------------------------------- |
 | `collapsible` | Boolean | `false` | Enables/disables the collapsible functionality.                                   |
+| `label`       | String  |         | Caption for the fieldset.                                                         |
 | `opened`      | Boolean | `false` | Initial collapsible state. Applied when the collapsible functionality is enabled. |
 
+## Examples
+
```
Comment (rogyar) [inline guides/v2.2/ui_comp_guide/components/ui-fieldset.md]: Please, add another subheader that explains the following example. Refer the [following](https://github.com/magento/devdocs/pull/6039/files#diff-4204d17776bb584fd954af5edac31c25R28) PR as an example.

Thank you

File: guides/v2.2/ui_comp_guide/components/ui-fieldset.md [hunk 2]
```diff
@@ -20,8 +20,46 @@ The Fieldset component implements a container for visually-grouped form elements
...
+The following example adds a new main fieldset, that contains a collapsible fieldset.
```
Comment (keharper) [inline guides/v2.2/ui_comp_guide/components/ui-fieldset.md]:
```suggestion
The following example adds a new main fieldset that contains a collapsible fieldset.
```

File: guides/v2.2/ui_comp_guide/components/ui-fieldset.md (hunk 1 above)
Comment (eduard13) [reply guides/v2.2/ui_comp_guide/components/ui-fieldset.md]: Hi @rogyar, the subheader has been added.
Thank you.

File: guides/v2.2/ui_comp_guide/components/ui-fieldset.md (hunk 2 above)
Comment (eduard13) [reply guides/v2.2/ui_comp_guide/components/ui-fieldset.md]: Done.
Thank you.

**Comment by keharper** [comment]
running tests

**Pull request merged**